cabal-install-solver/src/Distribution/Solver/Modular/Message.hs

```diff
@@ -279,6 +279,7 @@ showFR :: ConflictSet -> FailReason -> String
 showFR _ (UnsupportedExtension ext)       = " (conflict: requires " ++ showUnsupportedExtension ext ++ ")"
 showFR _ (UnsupportedLanguage lang)       = " (conflict: requires " ++ showUnsupportedLanguage lang ++ ")"
 showFR _ (MissingPkgconfigPackage pn vr)  = " (conflict: pkg-config package " ++ prettyShow pn ++ prettyShow vr ++ ", not found in the pkg-config database)"
+showFR _ (MissingPkgconfigProgram pn vr)  = " (pkg-config package " ++ prettyShow pn ++ prettyShow vr ++ " is needed but no pkg-config executable was found or querying it failed)"
```

Comment:
Should this actually be prefixed with `(conflict: ` as the other errors?

Reply:
I don't see it as a conflict between two things. It is just a requirement that cannot be satisfied. I suggest we remove `conflict:` from the `MissingPkgconfigPackage` case on the previous line too.

The other are dubious too. Perhaps `UnsupportedExtension` should read:
```
(unavailable language extension: " ++ showUnsupportedExtension ext ++ ")"
```
but we should revisit the non-pkg-config related cases later.